### DISCUSSION TITLE
Hubs on Azure?

### DISCUSSION THREAD

**Discussion by JeffreyConnected** [mozilla/hubs#2705] [Hubs Cloud Questions] [answered]
Are there any plans to publish instructions on deploying Hubs to Azure? Has anyone ever deployed Hubs to Azure?

**Comment by misslivirose** [comment, marked as answer]
Hi @JeffreyConnected - we don't have a marketplace listing for Azure for Hubs Cloud the way that we do for AWS, but it's something we're considering for the future. I don't know of anyone having set up Hubs on Azure specifically at the moment, but there is a bit more information about doing a DIY setup for Hubs in #2699, which may be able to be adapted for Azure.
